src/libs/Navigation/Navigation.js

```diff
@@ -125,13 +125,22 @@ function dismissModal(shouldOpenDrawer = false) {
 /**
  * Check whether the passed route is currently Active or not.
  *
+ * Previous implementation used navigationRef.current.getCurrentRoute().path, which was
+ * undefined in the first navigation. Hence, in our current solution, we're rebuilding
+ * the active route using the useLinkBuilder hook that takes routeName and params as the arguments.
+ *
```

Comment:
I don't think mentioning the previous and current implementation is necessary. Perhaps something like this would be enough: 

`Building path with useLinkBuilder since navigationRef.current.getCurrentRoute().path is undefined in the first navigation.`

Reply:
Okay

Reply:
Done